client/src/components/Search/index.js

```diff
@@ -34,15 +36,52 @@ export default class Search extends Component {
 
   handleSubmit(event) {
     // TODO - hookup to map instance and add marker for given location
-    alert('A query was submitted: ', this.state.value);
     event.preventDefault();
+    axios.post(`${API_SERVER}/search/textsearch`, {
+      input: this.state.value,
+      lat: this.props.position.lat,
+      lng: this.props.position.lng,
+      }).then((response) => { 
+        if(response.data.status == OK_Status)
+        {
+          let [searchResult] = response.data.results;
```

Comment:
this should be changed to 'const' since `searchResult` is never modified within this score

Reply:
We can create a separate issue for this if it seems tricky to figure out but I ran into an instance were the marker generated for a location in another state. It didn't send me to the location of the first prediction as in other cases. I think we may need to be careful in assuming the first data in the textSearch result array maps is based on proximity to user location.

Reply:
As per the doc, it states "If you specify a location parameter, you must also specify a radius parameter. radius — Defines the distance (in meters) within which to bias place results. The maximum allowed radius is 50 000 meters. Results inside of this region will be ranked higher than results outside of the search circle; however, prominent results from outside of the search radius may be included." 
We don't have a value specified for radius as part of request currently. Do you think that would help resolve the issue you faced? Shall we default radius to certain value?

Reply:
@khusbuchandra yeah that is probably the best solution. I'll read up more on the specs for text search service. Please create an issue for this for milestone MVP 2.0 as a reminder of something to dig into further.

Depending on how far we go with this app and its feature, we may dictate the radius depending on mode of transportation selected; we probably want different limits for walking vs biking vs driving.